openbsd-games.db

```diff
@@ -5834,7 +5834,7 @@ Cover	A_Virus_Named_TOM_-_cover.jpg
 Engine	FNA
 Setup	fnaify
 Runtime
-Store	https://www.humblebundle.com/store/a-virus-named-tom https://www.humblebundle.com/store/a-virus-named-tom-with-soundtrack
+Store	https://www.humblebundle.com/store/a-virus-named-tom https://www.humblebundle.com/store/a-virus-named-tom-with-soundtrack https://store.steampowered.com/app/207650
```

Comment:
This here doesn't run past the first level. Would discourage purchasing this one. I have looked for the source of the problem for a long time, to no avail.